com.itopia.client.appdata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.itopia.client</id>
+  <name>itopia Remote Desktop Client</name>
+  <summary>A Client for Remote Desktop Protocol (RDP) Servers</summary>
+  <developer_name>itopia, Inc.</developer_name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://www.itopia.com/terms-and-conditions/</project_license>
+
+  <url type="homepage">https://www.itopia.com/</url>
+
```

Comment:
The appdata file lacks some screenshots, OARS tag and a description. You can read more about the requirements at https://github.com/flathub/flathub/wiki/AppData-Guidelines

Reply:
I've added the requested screenshots and description